### DISCUSSION TITLE
Welcome to vnote Discussions!

### DISCUSSION THREAD

**Discussion by tamlok** [tamlok/vnote#1592] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arvindsridharan** [comment]
Hi, This is Arvin.  I am excited to be part of the Git gub group.  Everyday when I see new products being rolled out in the GIt Hub space, I wonder when I will be able to build some awesome product like Vnote.  I have started my Javascript journey with HTML and CSS.  Looking forward to ideas and inspiration.  Thanks.
